### ISSUE TITLE
image patch issue, "&#37;5C"

### ISSUE THREAD

**Issue by prikhi** [prikhi/pencil#641]
_From [hong.res...@gmail.com](https://code.google.com/u/102484687111103611655/) on June 26, 2014 02:02:36_

in windows environment export, then update load to linux server, "&#37;5C" was not recognized by linux server.

I just added a single jquery to work around this issue:
1, open StyleSheet.xslt
added this to the header part
# 

$("img[src]").each(function(){$(this).attr("src",$(this).attr("src").replace('&#37;5C','/'));});
# 

for I'm using jquery template, so final will be
# 

```
            \<SCRIPT type="text/javascript">
              $(document).ready(function () {
              $('body').layout({ applyDefaultStyles: true,
              minSize:140 });
              $("img[src]").each(function(){$(this).attr("src",$(this).attr("src").replace('&#37;5C','/'));});
              });
            \</SCRIPT>
```
# 

works

_Original issue: http://code.google.com/p/evoluspencil/issues/detail?id=641_

**Comment by prikhi** [comment]
_From [hong.res...@gmail.com](https://code.google.com/u/102484687111103611655/) on June 30, 2014 01:43:28_

sorry, forgot to post my fixed version of template which I said just in my last post

**Attachment:** [JQUERY_Template.zip](http://code.google.com/p/evoluspencil/issues/detail?id=641#hc1)

**Comment by prikhi** [comment]
Fixed by #737
